helm-charts/splunk-otel-collector/templates/operator/_helpers.tpl

```diff
@@ -138,7 +143,7 @@ Helper for generating environment variables for each instrumentation library.
       {{- $otelResourceAttributes = printf "%s,%s" $env.value $otelResourceAttributes }}
     {{- else }}
       {{- printf "- name: %s" $env.name | nindent 6 -}}
-      {{- printf "  value: %s" $env.value | nindent 6 -}}
+      {{- printf "  value: \"%s\"" $env.value | nindent 6 -}}
```

Comment:
Unrelated fix?

Reply:
Anyway, `quote` should be used instead to avoid double-quoting.

Reply:
```yaml
        value: {{ quote $env.value }}
```

Reply:
Done